scss/components/_Flash.scss

```diff
@@ -0,0 +1,35 @@
+// default component vars listed at the top of the component stylesheet
+// those vars are transferred to rev-settings AND are commented out
+// this way updates can be made to a component by uncommenting settings vars
+$flash-background: $primary !default;
+$flash-border-size: 1px !default;
+$flash-border-color: $divider-color-dark !default;
+$flash-color: $white !default;
+$flash-padding: $global-vertical-space $global-horizontal-space !default;
+
+.rev-Flash {
+  @include color-management;
+  @include typography-contained;
+  background: $flash-background;
+  border: $flash-border-size solid $flash-border-color;
+  color: $flash-color;
+  margin-bottom: $global-margin;
+  padding: $global-padding-tiny 0;
+  position: relative;
+
+  &.rev-Flash--fade {
+    @include fade-out--hide();
+  }
+  &.rev-Flash--dismissible {
```

Comment:
I would maybe even add a `padding-right: $flash-dismissible-padding-right` so that the text doesnt overlap the close button. but I need to see what it looks like with the close button first. currently not seeing one on your branch.

if so, don't forget to add it to the settings-template